tests/Unit/API/Site/Controllers/Ads/RecommendationsControllerTest.php

```diff
@@ -119,23 +119,6 @@ public function test_get_recommendations_returns_empty_array_when_filter_by_non_
 		$this->assertEquals( 'rest_invalid_param', $response->get_data()['code'] );
 	}
 
-	public function test_get_recommendations_returns_error_if_account_not_connected() {
```

Comment:
@mukeshpanchal27 is there a way we can still test for the error response if account is not connected?